spec/controllers/tasks_controller_spec.rb

```diff
@@ -114,8 +114,12 @@
     it 'can show offenders needing nDelius updates' do
       stub_offender(offender_no, booking_number: 754_206)
 
```

Comment:
unsure about this change - it seems unrelated to the story. Concerned that we are changing the sense of a test, and it passes by accident than by design.

It is possible to have a nil team when probation service is NI or Scotland, so maybe that might have been a better change to make here?